application/web/main.css

```diff
@@ -103,6 +115,7 @@ button.new-journal img.journal-icon {
     align-items: center;
     cursor: pointer;
     font-size: 16px;
+    margin-bottom: 10px;
```

Comment:
I committed a breaking change for this potentially.

Reply:
When I pulled I saw the line 'min-height: 40px;' but I dont notice the change that puts space between the my journals
![image](https://github.com/cse110-sp24-group3/cse110-sp24-group3/assets/52176650/e1c1bbe9-3b62-42e0-a2fa-e83bdb23044b)